reqMgrClient.py

```diff
@@ -399,8 +427,8 @@ def getWorkflowInfo(url, workflow):
     """
```

Comment:
Julian, could you give me that commends or script you are using for the testing? It sounds like there is accept header not set for this. Anyway, I think it is easier if I can run the test. Thanks